xdebug/xdebug.php

```diff
@@ -189,9 +189,9 @@ function xdebug_break(): bool {}
  * Use the PHP function getcwd() to figure out what the current working directory is.
  * @param string|null $traceFile
  * @param int $options
- * @return string
+ * @return string|null
  */
-function xdebug_start_trace(?string $traceFile = null, int $options = 0): string {}
+function xdebug_start_trace(?string $traceFile = null, int $options = 0): ?string {}
```

Comment:
test failed, so i adjusted it accordingly ([doc](https://xdebug.org/docs/all_functions#xdebug_start_trace))

Reply:
Good catch @nick-zh! 👍  But actually, I've just pushed the same fix (+ related phpdoc change) so could you please rebase your branch on master and force push it after that?